File: SoloLearn/Roadrunner/Roadrunner.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int distance = Convert.ToInt32(Console.ReadLine()), runnerSpeed = Convert.ToInt32(Console.ReadLine()), coyoteSpeed = Convert.ToInt32(Console.ReadLine());
            double runnerTime = distance / runnerSpeed, coyoteTime = (distance + 50) / coyoteSpeed;
            if (runnerTime < coyoteTime)
                Console.WriteLine("Meep Meep");
            else
                Console.WriteLine("Yum");
        }
    }
} 

File: SoloLearn/Roadrunner/Roadrunner.swift
if let distance = readLine(), let runnerSpeed = readLine(), let coyoteSpeed = readLine(), let d = Int(distance), let rs = Int(runnerSpeed), let cs = Int(coyoteSpeed) {
    let runnerTime = d / rs
    let coyoteTime = (d + 50) / cs
    if runnerTime < coyoteTime {
        print("Meep Meep")
    } else {
        print("Yum")
    }
}

